<?xml version="1.0"?>

<launch>
  <node pkg="icl_multiple_stationary" type="odom_main_node" name="odom_main_node">

	  <param name="bodyName" value=""/>
    <param name="cameraName" value="camera"/>

    <param name="useMocap" value="true"/>

    <param name="pcbx" value="0.105638"/>
	  <param name="pcby" value="0.0702751"/>
	  <param name="pcbz" value="0.28"/>

    <param name="qcbw" value="0.722352"/>
    <param name="qcbx" value="-0.691162"/>
    <param name="qcby" value="0.0217651"/>
    <param name="qcbz" value="0.00289808"/>

    <!-- cutoff frequencies -->
    <param name="fp" value="5.0"/>
    <param name="fq" value="5.0"/>
    <param name="fv" value="50.0"/>
    <param name="fw" value="50.0"/>

  </node>

  <!-- <node pkg="odom_estimator" type="odom_node" name="odom_node_cam" output="screen">
    <rosparam param="names"> ["turtlebot3/camera"] </rosparam>
  </node> -->

</launch>
